gwctl/pkg/printer/gatewayclasses.go

```diff
@@ -89,11 +100,34 @@ func (gcp *GatewayClassesPrinter) PrintDescribeView(resourceModel *resourcedisco
 	index := 0
 	for _, gatewayClassNode := range resourceModel.GatewayClasses {
 		index++
+		apiVersion, kind := gatewayClassNode.GatewayClass.GetObjectKind().GroupVersionKind().ToAPIVersionAndKind()
+		metadata := gatewayClassNode.GatewayClass.GetObjectMeta()
 
+		// views ordered with respect to https://deploy-preview-2723--kubernetes-sigs-gateway-api.netlify.app/geps/gep-2722/#:~:text=gwctl%20describe%20gatewayclass%20foo%2Dcom%2Dexternal%2Dgateway%2Dclass
 		views := []gatewayClassDescribeView{
 			{
 				Name: gatewayClassNode.GatewayClass.GetName(),
 			},
+			{
+				Labels: ptr.To(gatewayClassNode.GatewayClass.GetLabels()),
+			},
+			{
+				Annotations: ptr.To(gatewayClassNode.GatewayClass.GetAnnotations()),
+			},
+			{
+				APIVersion: apiVersion,
+			},
+			{
+				Kind: kind,
+			},
+			{
+				Metadata: &metav1.ObjectMeta{
```

Comment:
Lets follow a consistent approach as: https://github.com/kubernetes-sigs/gateway-api/blob/877803e258c71249bd863246da399055ba1e6eee/gwctl/pkg/printer/policies.go#L176-L180
where we remove duplicated fields from metadata and print the rest? (This would allow for possible future expansion of the fields within Metadata to be included by default)

Reply:
Sure, sounds neat